src/RouterConfig.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Navbar, Nav, NavItem, NavDropdown, MenuItem, FormGroup, Form, FormControl, Grid, Row, Col, Button } from 'react-bootstrap'
+import { Link, Switch, Redirect, Route, BrowserRouter } from 'react-router-dom'
+import { LinkContainer } from 'react-router-bootstrap'
+import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css'  // temporary, we can create our css later
+import AppointmentHome from './Appointment/AppointmentHome'
+import ServiceHome from './Service/ServiceHome'
+import RequestHome from './Request/index'
+import CreateRequest from './Request/CreateRequest';
+
+class RouterConfig extends React.Component {
+    render() {
+        return (
+            <BrowserRouter>
+            <Switch>
+              <Route exact path='/' render={() =>
+                  <header className="App-basic">
+                      <p>
+                        Body Contouring Clinic <br/>
+                        React : true <br/>
+                        {`Express : ${this.state.backServer}`}
```

Comment:
This causes error.
`this.state` is not set in this file. It is still in the App.js file

Reply:
Yeah, I was fixing this one. I will change it tomorrow morning.